### DISCUSSION TITLE
The website djangonaut.space Images and Icons seems broken (2nd January 2024)

### DISCUSSION THREAD

**Discussion by dostogircse171** [djangonaut-space/program#22] [General]
Hi,
I was checking [https://djangonaut.space/](https://djangonaut.space/) and found the footer icons and events page images are broken.

Also, "Code of Conduct" in the footer link need to be updated it is going 404 as well.

I wanted to let you know about this. 
I couldn't find any repository for the website otherwise I would love to fix these small changes. 

<img width="902" alt="Screenshot 2024-01-03 at 7 16 58 pm" src="https://github.com/djangonaut-space/program/assets/28702192/6c0076a8-7c23-43b6-af83-5dbeba0a43ef">
<img width="1697" alt="Screenshot 2024-01-03 at 7 17 25 pm" src="https://github.com/djangonaut-space/program/assets/28702192/fe50996a-bf6e-4fb7-ac52-d45526e08e5f">

Best,
Golam

**Comment by tim-schilling** [comment]
Thanks Golam! Our site is managed by this repo https://github.com/djangonaut-space/wagtail-indymeet. We're aware of both issues. The icons is a CORS issue: https://github.com/djangonaut-space/wagtail-indymeet/issues/143 The media files is something we've "solved", but need to reupload some images.
